tests/jerry/es2015/math-expm1.js

```diff
@@ -0,0 +1,26 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var p_zero = 0.0;
+var n_zero = -p_zero;
+
+assert(isNaN(Math.expm1(NaN)));
+assert(Math.expm1(p_zero) === p_zero);
+assert(Math.expm1(n_zero) === n_zero);
```

Comment:
Comparing to -0.0 won't work with == or ===, because 0.0 will be equal to -0.0.

Previously I found and fixed a similar bug, you can see 
the proper compare in tests/jerry/es2015/math-trunc.js .

Reply:
Thanks, you are right, it helped me to find bug in source code ;)